README.md

```diff
@@ -43,98 +43,54 @@ Both options require you generate an ssl certificate & key via openssl:
 
 # Features
 
-## General Starting CSS
+## Familiar 2D UI API
 
-*For now there is some needed initial css for readability of the following feature examples*
-*Play around with the options as you get more used to the 2D/3D html setup*
+Create 2D UI using CSS and `mr-panel`
 
-```css
-* {
-    padding: 0;
-    margin: 0;
-    border: none;
-    border-collapse: collapse;
-}
-
-html {
-    overflow: hidden;
-    overscroll-behavior: none;
-}
-
-body {
-    position: fixed;
-}
-
-mr-container {
-    height: 100vh;
-    width: 100%;
+```html
+<style>
+.layout {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+    gap: 10px;
+    grid-auto-rows: minmax(100px, auto);
 }
+.title {
+    margin: 0 auto;
+    font-size: 5vw;
+    line-height: 100%;
+    color: rgba(24, 24, 24, 0.75);
 
-mr-app * {
-    display: block;
+    grid-column: 2;
 }
 
-mr-text {
-    color: red;
-    font-size: 100px;
+mr-img {
+    object-fit: cover;
+    grid-row: 3 / 6;
+    grid-column: 1 / -1;
 }
 
-mr-model {
-    scale: 0.001;
-    z-index: 100;
+#logo {
+    grid-column : 2;
+    scale: 0.001; /* set 3D content size */
+    z-index: 100; /* set position on Z-axis */
 }
-
-```
-
-## 2D UI & Layout Components
-
-```html
+</style>
 <mr-app>
-    <!-- The 3D Area -->
-    <mr-surface>
-        <!-- The 3D UI Container -->
-        <mr-container>
-            <mr-row>
-                <mr-text>
-                    This is a quick example of an image gallery with explainer text.
-                </mr-text>
-                <mr-column>
-                    <mr-img src="..."></mr-img>
-                    <mr-row height="0.02">
-                        <mr-button onClick="Prev()"> <- </mr-button>
-                        <mr-button onClick="Next()"> -> </mr-button>
-                    </mr-row>
-                </mr-column>
-            </mr-row>
-        </mr-container>
-    </mr-surface>
+    <!-- The 2D UI Panel -->
```

Comment:
could we also put one element that is 3d outside of the panel for the readme demo? ik we have included the mr-model elemnt on line 88, just a bit concerned of using just the naming of 2D though

Reply:
ooo good catch. I might add this in another PR with the button/img update. I'd like it to be in the demo page as well.